doc/ref_model/chapters/chapter02.md

```diff
@@ -266,17 +266,44 @@ By trying to sort workloads into different categories based on the requirements
 <a name="2.4"></a>
 ## 2.4 Cloud Infrastructure Profiles
 
-Based on the above analysis, following cloud infrastructure profiles are proposed (also shown in **Figure 2-1** below)
+Infrastructure profiles are used to tag infrastructure (such as hypervisor hosts, or Kubernetes worker nodes) and associate it with a set of capabilities exploitable by the workloads.
+Two profile layers are proposed:
+
+- The top level **profiles** represent macro-characteristics that partition infrastructure into separate pools, ie: an infrastructure object can belong to one and only one profile, and workloads can request to be instantiated on one infrastructure profile only.
+- Within each profile, **flavours** represent small deviations from the profile baseline that do not require partitioning the infrastructure into separate pools, whose specifications have a finer granularity than top-level profiles, and can be *optionally* requested by workloads that want a more granular control over what infrastructure they can run on, ie: an infrastructure object can have **more than one flavour** attached to it, and workloads can request to be instantiated infrastructure with a number of (0+) certain flavours.
+
+> Note: the fact that profiles do partition the infrastructure means that workloads can only choose one profile to run on, and infrastructure objects with a given profile will host only workloads requiring that specific profile.
+> 
+> On the other hand, the fact that flavours do not partition the infrastructure may mean that infrastructure objects with certain flavours may host workloads that do not require that flavour, but can run on that infrastructure partition anyway.
+
+
+### 2.4.1 Infrastructure profiles (top-level)
+
+Based on the above analysis, the following cloud infrastructure profiles are proposed (also shown in **Figure 2-1** below)
```

Comment:
Should we change that we support two different profiles with differing performance characteristics: Basic Performance and High Performance profiles? 

Change Network Intensive to either "High Performance" or "Performance Insensitive"?

Reply:
agree, i'll change to High Performance

Reply:
I agree with "High Performance"